Create ui-c8y-1018-503-168-correct-custom-navigator-logo-height-and-tabs-background-color-9539.md (#3544)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-168-correct-custom-navigator-logo-height-and-tabs-background-color-9539.md

---
date: ""

> title: Custom navigator logo height and tabs background color corrected

Reviewer: ```suggestion
title: Custom navigator logo height and tabs background color now displayed correctly
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63705
version: 1018.503.168
---

> correct custom navigator logo height and  tabs background color (#9539)

Reviewer 2: ```suggestion
Previously, the custom navigator logo height and tabs background color were not displayed correctly which led to inconsistent branding and poor user experience. With this change, the logo height and background color of the tabs in the custom navigator now render as intended.
```

> Previously, the custom navigator logo height and tabs background color were not displayed correctly which led to inconsistent branding and poor user experience. With this change, the logo height and background color of the tabs in the custom navigator now render as intended.

Reviewer: ```suggestion
Previously, the logo height and tabs background color in the custom navigator were not displayed correctly which led to inconsistent branding and poor user experience. With this change, the logo height and background color of the tabs in the custom navigator now render as intended.
```
